doc/code/qml_qnn.rst

```diff
@@ -0,0 +1,11 @@
+qml.qnn
+=======
+
+.. currentmodule:: pennylane.qnn
+
+.. automodapi:: pennylane.qnn
+	:no-heading:
+    :include-all-objects:
+    :no-inheritance-diagram:
+    :no-inherited-members:
+    :skip: CORRECT_TF_VERSION, INPUT_ARG, Iterable, Layer, Optional
```

Comment:
![image](https://user-images.githubusercontent.com/49409390/76650847-c6bc6c80-6539-11ea-9c67-48dda347f6d3.png)

It would be great to get rid of the empty "Variables" section, any thoughts?

Reply:
This page is rendered by the `automodapi` extension; you can hide the `Variables` section just by removing the `:include-all-objects:` flag above